login/core/validations/user_validation/DbValidation.js

```diff
@@ -0,0 +1,22 @@
+const Sequelize = require('sequelize');
+
+const SequelizeOperators = Sequelize.Op;
+const UserModel = require('../../../db/models/user_models/UserModel.js');
+const createUserAccount = require('../../../routers/controllers/user_controllers/UserServices.js');
+
+function CopyCheck(registrationRequest, response) {
```

Comment:
Name's not clear, this is registration, right? So why don't just name it Register/SignUp/whatever you find appropriate(but you have to be consistent, so it's the same on the frontend as well, don't mix up those definitions, pick one and stick to it)

Reply:
You are right,this file was not necessary
My war with git create it in my local branch
Thank you